### ISSUE TITLE
[EN] [Dungeon-Classic] [Scholomance] Got transformed into a heap of bones - permanently.

### ISSUE THREAD

**Issue by Tj20** [Atlantiss/BugTracker#1661]
http://www.wowhead.com/zone=6066/scholomance

During a Scholomance run something strange happened: in the first few rooms everything was [fine](http://i.imgur.com/r9eeMFT.jpg), but when I (the tank) tried to clear the [room under the entrance area](http://i.imgur.com/gAvmaSZ.jpg) (the one which contains many many mobs), I got transformed into a [pile of bones](http://i.imgur.com/Ie50AQ4.jpg).

I continued the run. After a wipe, I was now a [bigger heap of bones and flesh](http://i.imgur.com/qqX1oDG.jpg).

After the run I've tried everything to transform back: dying/rezzing, /script LeaveParty(), /script RepopMe(), getting polymorphed by a friendly mage in a duel... nothing worked. (At the moment I'm waiting for a GM to answer my ticket)

![booooones](http://i.imgur.com/nGUwc0i.jpg)

**Comment by Wolffenstein** [comment]
It's not a bug, it's a feature! :D

**Comment by Tj20** [comment]
Yeah, I've already contemplated doing some BGs now... at least I would be able to keep a low profile. :+1:

**Comment by xolon** [comment]
epic

**Comment by Wojmistrz** [comment]
don't fix that :D it's awesome MUST try this one :D

**Comment by Tj20** [comment]
Yeah, I did quests in the western plaguelands, however I abandoned any quests and also doesn't seem to have a related quest item in my inventory. Still I can't right-click the buff nor does it have a duration. Strange.

**Comment by Howlingwolf** [comment]
New playable race I guess. Carcass. Have you tried change talent specialization? It prowides partially unaura, so it could help.

**Comment by Tj20** [comment]
yeah, didn't work either, but meanwhile a GM answered my ticket and helped me.

**Comment by Varchord** [comment]
That's awesome!

**Comment by Tj20** [comment]
I just noticed [Scholomance Necromancers](http://cata.openwow.com/npc=10477#abilities) cast [Corpse Explosion](http://cata.openwow.com/spell=17616).

Maybe one of those was involved. 

Maybe I died, they cast corpse explosion on me, then I was ressed again, but the transformation was permanent.

**Comment by Aival94** [comment]
i tried to search a little and found this one.I am not sure if this mob even spawn on scholomance as is a random spawn but if you remember him being there this might cause your issue as have two summon spells with script effect.

Wowhead link: [Kormok](http://www.wowhead.com/npc=16118/kormok)
Wowhead spells with script effect links: [Summon Bone Mages](http://www.wowhead.com/spell=27695/summon-bone-mages) and [Summon Bone Minions](http://www.wowhead.com/spell=27687/summon-bone-minions)

Gamepedia link: [Kormok](http://wow.gamepedia.com/Kormok)

**Comment by Whiteguardian** [comment]
And on February 13, he came back ...
Well hello everyone first.
I also happened to the same situation last February 13, I'm just writing here for a GM to help me because the bug has no solution without GM's help.
But yeah, it sure is funny to see a lot of bones tanking one boss.

**Comment by Whiteguardian** [comment]
@Piklo 
I already made the ticket in the game. I've been waiting for four days.
